pkgs/applications/window-managers/i3/workstyle.nix

```
@@ -5,7 +5,7 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "workstyle";
-  version = "0.2.1";
+  version = "0.2.1-p0";
```

```suggestion
  version = "0.2.1";
```
We don't do such things.

---

Noted.